app/presenters/simple_smart_answer_presenter.rb

```diff
@@ -17,6 +17,8 @@ def start_button_text
 
     if details["start_button_text"] == "Start now"
       I18n.t("formats.start_now")
+    elsif detals["start_button_text"] == "Continue"
```

Comment:
As far as solutions go this is miserable, however this appears to be how we handle the labelling here. https://github.com/alphagov/frontend/commit/a6e9db442ada1a983ae47818075776f6c8bcf70a